deploy/backplane/mtsre/10-mtsre-admins-project.ClusterRole.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-admins-project
+rules:
+# SRE can get, list, and watch all resources
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+# SRE can portforward pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/portforward
+  verbs:
+  - create  
+# SRE can manage addons namespace
+- apiGroups:
+  - "addons.managed.openshift.io"
+  resources:
+  - "addonoperators"
+  - "addons"
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+  - update
+  - create
```

Comment:
This is not a namespace scoped resource

Reply:
@sandhyadalavi If you do need the perms, you can specify the `resourceNames` in the rbac rule, here is an [example](https://github.com/openshift/managed-cluster-config/blob/8e1d9887a8fafd226641ed0aa73fadc0dabe4ca1/deploy/osd-customer-monitoring/05-prometheus-k8s-role.yaml#L11), wdyt @cblecker ?

Reply:
@wanghaoran1988 @cblecker we want to manage resources for addon-operator in openshift-addon-operator namespace